README.md

```diff
@@ -18,6 +18,8 @@ Lotus is an implementation of the Filecoin Distributed Storage Network. For more
 
 ## Building & Documentation
 
+> Note: The default `master` branch is the dev branch, please use with caution. For the latest stable version, checkout the most recent [release](https://github.com/filecoin-project/lotus/releases).
```

Comment:
Note that the most recent release in this list could be a pre-release ;-) Just playing devil's advocate.

Reply:
Thanks! updated